### PR TITLE
Added typing for typescript

### PR DESCRIPTION
**Pull request by Xample**
All is in the title

### CODE REVIEW

**Comment by Xample** [comment]
@dcousens the build seems to have failed on a [lint test](https://travis-ci.org/dcousens/haversine-distance/jobs/457360090#L661) on the js file I have not modified. Did this ever worked before?

**Comment by dcousens** [comment]
@Xample it is OK now, please rebase :+1: 

> Did this ever worked before?

I use `"standard": "*"`,  this means sometimes it breaks without any interaction (by design)

File: index.d.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    lat: number;
+    lon: number;
+}
+
+/**
+ * Return the Haversine distance in meters
+ * @param a first location
+ * @param b second location
+ */
+export default function haversineDistance(a: LocationUnion, b: LocationUnion): number;
```
Comment (dcousens) [inline index.d.ts:23]: space between `distance` and `(` probably

**Comment by Xample** [comment]
@dcousens rebased, it's in your hands

File: index.d.ts (hunk 1 above)
Comment (Xample) [reply index.d.ts:23]: I am using the one of, if not "the" default lint for ts but we can add a space if needed… note that it looks weird with that space
```suggestion
export default function haversineDistance (a: LocationUnion, b: LocationUnion): number;
```

**Comment by Xample** [comment]
@dcousens still no merge, I rebased and pushed a month ago, did I miss something?

**Comment by dcousens** [comment]
Thanks for the bump @Xample :+1: great work

**Pull request merged**